README.md

```diff
@@ -123,9 +123,9 @@ Run the next commands
 
 ## Run Cromwell Pipeline in the Docker (Local)
 
-- Change values of the environment variables in [docker.sbt](portal/docker.sbt) file
+- Create .env file with environment variables from .env.example file in the root folder
```

Comment:
*minor* Probably we should add this `.env` file to `.gitignore`

Reply:
You are right! Did it